docs/developer-docs/latest/development/using-providers.md

```diff
@@ -0,0 +1,204 @@
+
+# Install the Sendgrid provider for the Email plugin
+yarn add @strapi/provider-email-sendgrid --save
+```
+</code-block>
+
+</code-group>
+
+## Configuring providers
+
+Newly installed providers are enabled and configured in the `./config/plugins.js` file. If this file does not exist you must create it.
+
+Each provider will have different configuration settings available, review the respective entry for that provider in the [Marketplace](../../../user-docs/latest/plugins/installing-plugins-via-marketplace.md) or [NPM](https://www.npmjs.com/) to learn more.
+
+Below are example configurations for the Upload and Email plugins.
+
+:::: tabs card
+
+::: tab Upload
+
+```js
+// path: ./config/plugins.js
+
+module.exports = ({ env }) => ({
+  // ...
+  upload: {
+    config: {
+      provider: 'aws-s3', // For community providers pass the full package name (e.g. provider: 'strapi-provider-upload-google-cloud-storage')
+      providerOptions: {
+        accessKeyId: env('AWS_ACCESS_KEY_ID'),
+        secretAccessKey: env('AWS_ACCESS_SECRET'),
+        region: env('AWS_REGION'),
+        params: {
+          Bucket: env('AWS_BUCKET'),
+        },
+      },
+    },
+  },
+  // ...
+});
+```
+
+Strapi has a default Security Middleware that has a very strict `contentSecurityPolicy` that limits loading images and media to `"'self'"` only, see the example configuration on the [provider page](https://www.npmjs.com/package/@strapi/provider-upload-aws-s3) or the [middleware documentation](/developer-docs/latest/setup-deployment-guides/configurations/required/middlewares.md#loading-order) for more information.
+
+:::
+
+::: tab Email
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  email: {
+    config: {
+      provider: 'sendgrid', // For community providers pass the full package name (e.g. provider: 'strapi-provider-email-mandrill')
+      providerOptions: {
+        apiKey: env('SENDGRID_API_KEY'),
+      },
+      settings: {
+        defaultFrom: 'juliasedefdjian@strapi.io',
+        defaultReplyTo: 'juliasedefdjian@strapi.io',
+        testAddress: 'juliasedefdjian@strapi.io',
+      },
+    },
+  },
+  // ...
+});
+```
+
+Keep in mind that:
+
+* When using a different provider per environment, specify the correct configuration in `./config/env/${yourEnvironment}/plugins.js`. See [Environments](/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.md).
+* Only one email provider will be active at a time. If the email provider setting isn't picked up by Strapi, verify the `plugins.js` file is in the correct folder.
+* When testing the new email provider with those two email templates created during strapi setup, the _shipper email_ on the template defaults to `no-reply@strapi.io` and needs to be updated according to your email provider, otherwise it will fail the test. See: [Configure templates locally](/user-docs/latest/settings/configuring-users-permissions-plugin-settings.md#configuring-email-templates).
```

Comment:
Make sure to follow the style guide properly. We use the following formatting for links: "(see Environments)" (at the end of the sentence, before the full stop in that case).